01-import_and_filter.py

```diff
@@ -23,8 +34,11 @@
 from mne.parallel import parallel_func
 from mne_bids.read import reader as mne_bids_readers
 from mne_bids import make_bids_basename, read_raw_bids
+from mne_bids.config import BIDS_VERSION
+from mne_bids.utils import _write_json
```

Comment:
hmm ... can we use public functions only? Since mne-study-template is not quite a package, I fear it may give people ideas

Reply:
mmmh but I thought that users are not supposed to rummage through the individual steps anyways? ... I could of course just use 

```Python
    json_output = json.dumps(dictionary, indent=4)
    with open(fname, 'w') as fid:
        fid.write(json_output)
        fid.write('\n')
```

without being verbose and having `overwrite`

Reply:
That's true, okay to keep it in then